src/projectApplications.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs');
+const path = require('path');
+const {google} = require('googleapis');
+
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly'];
+const ALL_QUARTERS_ID = '16vmuNiN-8mij8EoEIwvQ_wde9DgF_0IWXfT_95SHfcc';
+
+const numOfQuarters = 14;
```

Comment:
can make this programatic? instead of making it a hard set, idea is that we just change the endQuarter date everytime we need info